### PR TITLE
css e commerce page

### PR DESCRIPTION
**Pull request by PhillipLam89**
image

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Yo dude! Your output differs from the intended output, please revise to make this as close as possible to the original example. For instance, your font family is not the font family requested, the color for your text is incorrect in multiple places, and the placement of the text in comparison to the example is off as well. Let us know if you have any questions on how to adjust this!

File: css-ecommerce-page/styles.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+.dollar {
+  background-color: rgb(253, 238, 240);
+  font-weight: bold;
+  font-size: 12px;
+}
+
+.dress2 {
+  padding-right: 4rem;
```
Comment (codingcodymiller) [inline css-ecommerce-page/styles.css:43]: What is all of the padding for the remaining CSS rules here doing for you? It is a lot of padding, and there is probably a better way 🤔

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
This looks so much better, man!! I do have a question on your method of styling this though:

File: css-ecommerce-page/styles.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  text-align: center;
```
Comment (thebearingedge) [inline css-ecommerce-page/styles.css:7]: Don't `text-align: center` like this, it's why you have so many random spacings for your prices.

File: css-ecommerce-page/styles.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  padding-right: 4rem;
+}
+
+.cost-item-4 {
+  padding-right: 6.5rem;
+}
+
+.tiffany {
+  padding-right: 4rem;
+}
```
Comment (thebearingedge) [inline css-ecommerce-page/styles.css:64]: These should not be necessary. Probably will be fixed by not centering all the text.

File: css-ecommerce-page/styles.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+.LV {
+  padding-right: 2rem;
+}
+
+.LV-2 {
+  padding-right: 100px;
+}
```
Comment (thebearingedge) [inline css-ecommerce-page/styles.css:72]: What is LV?

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**